api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -136,6 +136,11 @@ message Bootstrap {
     bool enable_deferred_creation_stats = 1;
   }
 
+  message GrpcAsyncClientManagerConfig {
+    // Optional field to set the expiration time for the cached gRPC client object. The default is 50s.
+    google.protobuf.Duration max_cached_entry_idle_duration = 1 [(validate.rules).duration = {gt {}}];
```

Comment:
Consider adding a minimal value that is much larger than 0.
I think that at least 5 seconds makes sense here, but willing to hear others thoughts.
FWIW, it is always possible decreasing this value in the future without breaking anyone, but increasing it may break some user.

Reply:
Good point, +1, and decreasing this value will not break in future.